modules/drive-utils.sh

```diff
@@ -0,0 +1,53 @@
+# Load kernel modules and mount the drive to a local folder inside the root
+# of current project
+driveutils_mount () {
+    echo "[ Loading the kernel modules ]"
+    modprobe -v nbd max_part=8
+
+    echo "[ Mounting the disk image ]"
+    qemu-nbd --connect=/dev/nbd0 $DRIVE_PATH
+    mkdir -v -p "$DRIVE_MOUNT_PATH"
+    until mount /dev/nbd0p1 "$DRIVE_MOUNT_PATH"; do 
```

Comment:
Mount function should be limited by number of tries to mount the drive and unload the modules if no success achieved.

Reply:
The current behavior is not correct and may cause a script-unprocessable state where kernel modules are loaded and qemu-nbd is connected, but the disk is not mounted.

Reply:
Resolved in c77d20bcf554fd9e63559dd004f2ed69ebacb2b8